atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -198,7 +199,15 @@ public Void call() {
     protected <T> Iterable<List<T>> partitionByCountAndBytes(final Iterable<T> iterable,
                                                              final int maximumCountPerPartition,
                                                              final long maximumBytesPerPartition,
-                                                             final String tablename,
+                                                             final TableReference tableRef,
+                                                             final Function<T, Long> sizingFunction) {
+        return partitionByCountAndBytes(iterable, maximumCountPerPartition, maximumBytesPerPartition, tableRef.getQualifiedName(), sizingFunction);
```

Comment:
this changed from getMinimumDurationToTraceMillis() (which was the bug in the old version)